website_apps_store/controllers/main.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2017-2018 BizzAppDev
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import base64
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.http_routing.models.ir_http import slug
+from odoo.addons.website.controllers.main import QueryURL
+from odoo.addons.website_sale.controllers.main import WebsiteSale, TableCompute
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+PPG = 20  # Products Per Page
+PPR = 4   # Products Per Row
+
+
+class WebsiteSaleCustom(WebsiteSale):
+
+    def _get_search_domain(self, search, category, attrib_values):
+        domain = request.website.sale_product_domain()
+        if search:
+            for srch in search.split(" "):
+                domain += [
+                    '|', '|', '|', '|', '|', '|', '|', '|',
+                    ('name', 'ilike', srch),
+                    ('technical_name', 'ilike', srch),
+                    ('description', 'ilike', srch),
+                    ('description_sale', 'ilike', srch),
+                    ('product_variant_ids.default_code', 'ilike', srch),
+                    ('product_variant_ids.attribute_value_ids.name',
+                     'ilike', srch),
+                    ('product_variant_ids.app_description_rst_html',
+                     'ilike', srch),
+                    ('product_variant_ids.app_author_ids.name', 'ilike', srch),
+                    ('product_variant_ids.app_summary', 'ilike', srch)]
+        if category:
+            domain += [('public_categ_ids', 'child_of', int(category))]
+
+        if attrib_values:
+            attrib = None
+            ids = []
+            for value in attrib_values:
```

Comment:
This loop, for some reason, looks strange... but i don't tested it.

Reply:
This PR scope is migration so the logic should not change without a reason (normally changes on Odoo base code, new API, changes in guidelines,...etc) but feel free to suggest improvements with comment code.

Basicly the module should work as the previous version